app/src/org/commcare/activities/CommCareWiFiDirectActivity.java

```diff
@@ -836,10 +820,7 @@ public void onFormsCopied(String result) {
 
     @Override
     public void updatePeers() {
-        Logger.log(TAG, "Wi-Fi direct peers updating");
-        mManager.requestPeers(mChannel, (PeerListListener)this.getSupportFragmentManager()
-                .findFragmentById(R.id.frag_list));
-
+        Log.d("SHIVAMMM", "Wi-Fi direct peers updating");

```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/b67bf14163879f9fb09208858b78318e65c45843/app/src/org/commcare/activities/CommCareWiFiDirectActivity.java#L823"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>